tests/beaker_tests/cisco_acl/test_ace.rb

```diff
@@ -13,307 +13,136 @@
-  manifest_props: "
-    action          => 'permit',
-    proto           => 'tcp',
-    src_addr        => '1.2.3.4 2.3.4.5',
-    src_port        => 'eq 40',
-    dst_addr        => '9.9.0.4/32',
-    dst_port        => 'range 32 56',
-  ",
-  resource:       {
-    'action'   => 'permit',
-    'proto'    => 'tcp',
-    'src_addr' => '1.2.3.4 2.3.4.5',
-    'src_port' => 'eq 40',
-    'dst_addr' => '9.9.0.4/32',
-    'dst_port' => 'range 32 56',
-  },
+  master:        master,
+  agent:         agent,
+  resource_name: 'cisco_ace',
 }
 
-tests['ipv4_seq_20'] = {
-  desc:           '1.2 ipv4 beaker_1 seq 20',
-  title_pattern:  'ipv4 beaker_1 20',
-  manifest_props: "
-    action          => 'deny',
-    proto           => 'tcp',
-    src_addr        => 'any',
-    dst_addr        => 'any',
-  ",
-  resource:       {
-    'action'   => 'deny',
-    'proto'    => 'tcp',
-    'src_addr' => 'any',
-    'dst_addr' => 'any',
+# Test hash test cases
+tests[:seq_5] = {
+  preclean:       'cisco_acl',
+  title_pattern:  'ipv4 beaker 5',
+  manifest_props: {
+    # 'remark' is a standalone property
+    remark: 'seq_5 remark'
   },
 }
 
-tests['ipv4_seq_30'] = {
-  desc:           '1.3 ipv4 beaker_1 seq 30',
-  title_pattern:  'ipv4 beaker_1 30',
-  manifest_props: "
-    action        => 'permit',
-    proto         => 'tcp',
-    src_addr      => '1.2.3.4 2.3.4.5',
-    src_port      => 'eq 40',
-    dst_addr      => '8.9.0.4/32',
-    dst_port      => 'range 32 56',
-    tcp_flags     => 'ack syn fin',
-    precedence    => 'flash',
-    established   => true,
-    packet_length => 'range 80 1000',
-    time_range    => 'my_range',
-    redirect      => 'Ethernet1/1,Ethernet1/2,port-channel1',
-  ",
-  resource:       {
-    'action'        => 'permit',
-    'proto'         => 'tcp',
-    'src_addr'      => '1.2.3.4 2.3.4.5',
-    'src_port'      => 'eq 40',
-    'dst_addr'      => '8.9.0.4/32',
-    'dst_port'      => 'range 32 56',
-    'tcp_flags'     => 'ack syn fin',
-    'precedence'    => 'flash',
-    'established'   => 'true',
-    'packet_length' => 'range 80 1000',
-    'time_range'    => 'my_range',
-    'redirect'      => 'Ethernet1/1,Ethernet1/2,port-channel1',
+tests[:seq_10_v4] = {
+  title_pattern:  'ipv4 beaker 10',
+  manifest_props: {
+    action:   'permit',
+    proto:    'tcp',
+    src_addr: '1.2.3.4 2.3.4.5',
+    dst_addr: '9.9.0.4/32',
   },
 }
 
-tests['ipv6_seq_10'] = {
-  desc:           '2.1 ipv6 beaker_6 seq 10',
-  title_pattern:  'ipv6 beaker_6 10',
-  manifest_props: "
-    action          => 'deny',
-    proto           => 'tcp',
-    src_addr        => 'any',
-    dst_addr        => 'any',
-  ",
-  resource:       {
-    'action'   => 'deny',
-    'proto'    => 'tcp',
-    'src_addr' => 'any',
-    'dst_addr' => 'any',
+tests[:seq_10_v6] = {
+  desc:           'IPv6 Seq 10',
+  title_pattern:  'ipv6 beaker6 10',
+  platform:       'n(3|7|9)k', # TBD: N5k addrs display as 0001:0001::0001/128
```

Comment:
I sent you the link for this.. should we go ahead and take care of it now or do you want to defer?